incubator/elasticsearch/README.md

```diff
@@ -0,0 +1,65 @@
+# Elasticsearch Helm Chart
+
+This image is using Fabric8's great work around the [kubernetes plugin](https://github.com/fabric8io/elasticsearch-cloud-kubernetes) for elasticsearch and their [image](https://hub.docker.com/r/fabric8/elasticsearch-k8s/) as parent.
+
+Credit to them!
+
+## Prerequisites Details
+
+* Kubernetes 1.3 with alpha APIs enable
+* PV dynamic provisioning support on the underlying infrastructure
+
+## PetSet Details
+* http://kubernetes.io/docs/user-guide/petset/
+
+## PetSet Caveats
+* http://kubernetes.io/docs/user-guide/petset/#alpha-limitations
+
+## Todo
+
+* Implement TLS/Auth/Security
+* Smarter upscaling/downscaling
+
+## Chart Details
+This chart will do the following:
+
+* Implemented a dynamically scalable elasticsearch cluster using Kubernetes PetSets/Deployments
+* Multi-role deployment: master, client and data nodes
+* PetSet Supports scaling down without degrading the cluster 
+
+## Get this chart
+
+```bash
+$ git clone https://github.com/kubernetes/charts.git
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release charts/incubator/elasticsearch
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the elasticsearch chart and their default values.
+
+|       Parameter       |           Description             |                         Default                          |
+|-----------------------|-----------------------------------|----------------------------------------------------------|
+| `Image`               | Container image name              | `jetstack/elasticsearch-pet`                             |
+| `ImageTag`            | Container image tag               | `2.3.4`                                                  |
+| `ImagePullPolicy`     | Container pull policy             | `Always`                                                 |
+| `ClientReplicas`      | Client node replicas (deployment) | `2`                                                      |
+| `MasterReplicas`      | Master node replicas (deployment) | `2`                                                      |
+| `DataReplicas`        | Data node replicas (petset)       | `3`                                                      |
+| `DataCpu`             | Data node requested cpu           | `250m`                                                   |
+| `DataMemory`          | Data node requested memory        | `2Gi`                                                    |
+| `DataStorage`         | Data persistent volume size       | `30Gi`                                                   |
+| `Component`           | Selector Key                      | `elasticsearch`                                          |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+# Deep dive
+
+TODO
```

Comment:
probably want to include the delete, since Petsets don't remove PV automatically.